src/core/server/deprecations/README.mdx

```diff
@@ -166,9 +194,13 @@ async function getDeprecations({ esClient, savedObjectsClient }: GetDeprecations
           }
       },
       manualSteps: [
-        'Using Kibana user management, change all users using the kibana_user role to the kibana_admin role.',
-        'Using Kibana role-mapping management, change all role-mappings which assing the kibana_user role to the kibana_admin role.'
-      ]
+        i18n.translate('security.deprecations.kibanaUserRole.manualStepOneMessage', {
+          defaultMessage: 'Using Kibana user management, change all users using the kibana_user role to the "kibana_admin" role.',
+        }),
+        i18n.translate('security.deprecations.kibanaUserRole.manualStepTwoMessage', {
+          defaultMessage: 'Using Kibana role-mapping management, change all role-mappings which assing the "kibana_user" role to the "kibana_admin" role.'
+        }),
```

Comment:
```suggestion
          defaultMessage: 'Update all mappings in Management > Security > Role Mappings to assign the kibana_admin role instead of the kibana_user role.'
        }),
```